### <font size=6>个人简介</font>
<p align="left"><font size="4" color=black>田侦，男，副教授，硕士生导师，中国计算机学会生物信息学专委会委员。2017年11月毕业于哈尔滨工业大学，获得计算机科学与技术专业博士学位。2018年3月至今在郑州大学计算机与人工智能学院软件工程系任教。主要讲授《微机原理及接口技术》《算法设计与分析》等课程，主要研究方向机器学习，数据挖掘和生物信息学。目前主持（完成）国家自然科学基金一项，河南省博士后基金一项，河南省科技攻关项目一项。目前已在Briefings in Bioinformatics, BMC Bioinformatics，Expert Systems with Applications ,TCBB, BMC Systems Biology和BIBM2020等学术期刊和会议上发表多篇论文，多次担任Briefings in Bioinformatics, TCBB和BIBM等期刊和会议审稿人。</font></p>

### <font size=6>教育与工作经历</font>

- <font size="4" color=black>2023年01月至今，郑州大学，计算机与人工智能学院，副教授</font>
- <font size="4" color=black>2020年01月-2022年12月，郑州大学，计算机与人工智能学院，副教授（直聘）</font>
- <font size="4" color=black>2020年01月-2021年02月，美国俄亥俄州立大学（Ohio State University），生物医学工程系，访问学者</font>
- <font size="4" color=black>2018年03月-2020年01月，郑州大学，信息工程学院，讲师</font>
- <font size="4" color=black>2013.09-2017.11：博士，哈尔滨工业大学，计算机科学与技术学院，计算机科学与技术专业（导师：郭茂祖）</font>
- <font size="4" color=black>2011.09-2013.07：硕士，哈尔滨工业大学，计算机科学与技术学院，计算机科学与技术专业</font>
- <font size="4" color=black>2007.09-2011.07：学士，哈尔滨工程大学，计算机科学与技术学院，计算机科学与技术专业</font>


###  <font size=6>研究方向</font>
- <font size="4" color=black>基因本体及基因功能相似度计算 </font>
- <font size="4" color=black>基于深度学习模型的生物实体关系预测</font>
- <font size="4" color=black>致病基因挖掘方法</font>
- <font size="4" color=black>知识图谱</font>


 
###  <font size=6>科研项目</font>

<font size="4" color=black>（一）主持科研项目</font>

- <font size="4" color=black>[1] 国家自然科学基金委员会，青年科学基金项目，61801432，基于多元数据融合的药物重定位方法研究，2019.01至2021.12，24万元，结题，主持</font>
- <font size="4" color=black>[2] 河南省博士后基金面上项目，247549，基于多元数据融合的药物-子通路关联关系预测，主持，结题</font>
- <font size="4" color=black>[3] 河南省科技攻关，32102211027，基于分层注意力机制的药物靶标关系预测研究，在研，主持</font>

<font size="4" color=black>（二）参与科研项目</font>

- <font size="4" color=black>[1] 国家自然科学基金委员会，面上项目，61671189，侯选药物筛选中分子对接及药靶网络构建方法，结题，参与</font>
- <font size="4" color=black>[2] 国家自然科学基金委员会，重点项目，61532014，基于网络模型的癌症相关模式挖掘理论与方法，结题，参与</font>
- <font size="4" color=black>[3] 国家自然科学基金委员会，面上项目，61402132，基于高通量测序数据多供体植物基因组结构变异识别方法研究，结题，参与</font>

### <font size=6>发表期刊论文列表</font>

<font size="4" color=black>（一）第一作者/通讯作者（*）</font>
- <font size="4" color=black> Zhixia Teng, Linyue Shi, Haihao Yu, Chengyan Wu, <b>Zhen Tian<sup>*</sup></b> . Measuring functional similarity of lncRNAs based on variable K-mer profiles of nucleotide sequences[J]. Methods, 2023, 212: 21-30.
- <font size="4" color=black> <b>Zhen Tian</b>, Yue Yu, Haichuan Fang, Weixin Xie, Maozu Guo<sup>*</sup>. Predicting microbe–drug associations with structure-enhanced contrastive learning and self-paced negative sampling strategy[J]. Briefings in Bioinformatics, 2023, 24(2): bbac634.</font>
- <font size="4" color=black><b>Zhen Tian<sup>#,*</sup></b>, Xiangyu Peng<sup>#</sup>（指导学生）, Haichuan Fang, Wenjie Zhang, Qiguo Dai, Yangdong Ye<sup>*</sup>. MHADTI: predicting drug–target interactions via multiview heterogeneous information network embedding with hierarchical attention mechanisms[J]. Briefings in Bioinformatics, 2022, 23(6): bbac434.</font>
- <font size="4" color=black> Zhengzheng Lou, Zhaoxu Cheng, Hui Li, Zhixia Teng, Yang Liu, <b>Zhen Tian<sup>*</sup></b>. Predicting miRNA–disease associations via learning multimodal networks and fusing mixed neighborhood information[J]. Briefings in Bioinformatics, 2022.</font>
- <font size="4" color=black> <b>Zhen Tian<sup>#</sup></b>, Haichuan Fang<sup>#</sup>（指导学生）, Zhixia Teng, Yangdong Ye<sup>*</sup>. GOGCN: Graph Convolutional Network on Gene Ontology for Functional Similarity Analysis of Genes. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2022.</font>
- <font size="4" color=black> Haichuan Fang（指导学生）, Youwei Wang, <b>Zhen Tian<sup>*</sup></b>, Yangdong Ye<sup>*</sup>. Learning knowledge graph embedding with a dual-attention embedding network[J]. Expert Systems with Applications, 2023, 212: 118806.</font>
-  <font size="4" color=black><b>Zhen Tian</b>, Haichuan Fang, Yangdong Ye, Zhenfeng Zhu<sup>*</sup>. A novel gene functional similarity calculation model by utilizing the specificity of terms and relationships in gene ontology[J]. BMC bioinformatics, 2022, 23(1): 1-14.</font>
-  <font size="4" color=black><b>Zhen Tian</b>, Zhixia Teng, Shuang Cheng, Maozu Guo<sup>*</sup>. Computational drug repositioning using meta-path-based semantic network analysis. BMC Systems Biology, 2018, 12(9): 123-134.</font>
- <font size="4" color=black> <b>Zhen Tian</b>, Haichuan Fang, Yangdong Ye, Zhenfeng Zhu<sup>*</sup>. SWE: a novel method with semantic-weighted edge for measuring gene functional similarity[C]. 2020 IEEE International Conference on Bioinformatics and Biomedicine (BIBM). IEEE, 2020: 1672-1678.</font>
- <font size="4" color=black><b>Zhen Tian</b>, Maozu Guo<sup>*</sup>, Chunyu Wang, Xiaoyan Liu, Shiming Wang. Tian Z, Guo M, Wang C, et al. Refine gene functional similarity network based on interaction networks[J]. BMC bioinformatics, 2017, 18(16): 183-193.</font>
- <font size="4" color=black> <b>Zhen Tian</b>, Maozu Guo<sup>*</sup>, Chunyu Wang, LinLin Xing, Lei Wang, Yin Zhang. Constructing an integrated gene similarity network for the identification of disease genes[J]. Journal of biomedical semantics, 2017, 8(1): 27-41.</font>
- <font  color=black><b>Zhen Tian</b>, Chunyu Wang, Maozu Guo<sup>*</sup>, Xiaoyan Liu, Zhixia Teng. An improved method for functional similarity analysis of genes based on Gene Ontology[J]. BMC systems biology, 2016, 10(4): 465-484.</font>
-   <font size="4" color=black> <b>Zhen Tian</b>, Chunyu Wang, Maozu Guo<sup>*</sup>, Xiaoyan Liu, Zhixia Teng. SGFSC: speeding the gene functional similarity calculation based on hash tables[J]. BMC bioinformatics, 2016, 17(1): 1-14. </font>
-  <font size="4" color=black> 朱真峰,<b> 田侦<sup>*</sup></b>. 基于思维导图的数据库原理课程总体认知分析[J]. 计算机教育, 2020.</font>

<font size="4" color=black>（二）共同作者</font>
- <font size="4" color=black>郭茂祖, 王诗鸣, 刘晓燕，<b>田侦</b> . miRNA 与疾病关联关系预测算法[J]. 软件学报, 2017, 28(11): 3094-3102.</font>
- <font size="4" color=black>Zhixia Teng, Maozu Guo, Xiaoyan Liu, <b>Zhen Tian</b>, Kai Che. Revealing protein functions based on relationships of interacting proteins and GO terms[J]. Journal of -biomedical semantics, 2017, 8(1): 9-17.</font>
- <font size="4" color=black>Zhixia Teng, Zhengnan Zhao, Yanjuan Li, <b>Zhen Tian</b>, Maozu Guo, Qianzi Lu, Guohua Wang i6mA-Vote: cross-species identification of DNA N6-methyladenine sites in plant genomes based on ensemble learning with voting[J]. Frontiers in Plant Science, 2022, 13.</font>
- <font size="4" color=black>Zhixia Teng, Zitong Zhang, <b>Zhen Tian</b>, Yanjuan Li, Guohua Wang. ReRF-Pred: predicting amyloidogenic regions of proteins based on their pseudo amino acid composition and tripeptide composition[J]. BMC bioinformatics, 2021, 22: 1-18.</font>
- <font size="4" color=black>Qiguo Dai, Zhaowei Wang, Jinmiao Song, Xiaodong Duan, Maozu Guo, <b>Zhen Tian</b>. A stacked ensemble learning framework with heterogeneous feature combinations for predicting ncRNA-protein interaction[C]//2020 IEEE International Conference on Bioinformatics and Biomedicine (BIBM). IEEE, 2020: 67-71.</font>
- <font size="4" color=black>Xiao Yang, <b>Zhen Tian</b>, Huayang Cui, Zhaoxin Zhang[C]. Link prediction on evolving network using tensor-based node similarity. 2012 IEEE 2nd International Conference on Cloud Computing and Intelligence Systems. IEEE, 2012, 1: 154-158.</font>



### <font size=6>培养学生</font>
- <font size="4" color=black>方海川（Haichuan Fang）2019级硕士</font>
- <font size="4" color=black>彭祥余（Xiangyu Peng）2020级硕士</font>
- <font size="4" color=black>余  跃（Yue Yu）2021级硕士</font>
- <font size="4" color=black>韩晨光（Chenguang Han）2021级硕士</font>
- <font size="4" color=black>徐乐文（Lewen Xu）2021级硕士</font>
- <font size="4" color=black>罗贤臣（Xianchen Luo）2022级硕士</font>
- <font size="4" color=black>张壮壮（Zhuangzhuang Zhang）2022级硕士</font>

### <font size=6>联系方式</font>
- <font size="4" color=black>电子邮箱：ieztian（AT）zzu.edu.cn  && zhentian_hit（AT）163.com</font>
- <font size="4" color=black>个人主页：http://www7.zzu.edu.cn/csai/info/1054/1397.htm</font>
- <font size="4" color=black>谷歌学术：https://scholar.google.com/citations?hl=zh-CN&user=6Eni8IgAAAAJ&view_op=list_works</font>
- <font size="4" color=black>联系地址：河南省郑州市科学大道100号，郑州大学计算机与人工智能学院，3313室</font>
- <font size="4" color=black>欢迎对机器学习、数据挖掘和生物信息处理等方向感兴趣的同学加入学习小组，并将个人简历和本科成绩单发送至联系邮箱。</font>


